components/UserInfos/UserInfos.tsx

```diff
@@ -74,9 +82,18 @@ const UserInfos: React.FC<UserInfosProps> = ({ lsp3JSON }) => {
           alt="profileImage"
         />
         <div className="ml-4">
-          <div className="text-xs">{addressFormatter(address)}</div>
+          {/* <div className="text-xs">{addressFormatter(address)}</div> */}
           <div className="text-2xl font-bold my-2">
-            {profile?.LSP3Profile.name}
+            {profile?.LSP3Profile.name ? (
+              <>
```

Comment:
do you really need the fragment here?

Reply:
I do need to check this condition but I created a function to do it above